python/ray/node.py

```diff
@@ -416,20 +462,25 @@ def start_raylet_monitor(self):
             process_info
         ]
 
+    def start_head_processes(self):
+        """Start head processes on the node."""
+        logger.info(
+            "Process STDOUT and STDERR is being redirected to {}.".format(
+                self._logs_dir))
+        assert self._redis_address is None
+        # If this is the head node, start the relevant head node processes.
+        self.start_redis()
+        self.start_monitor()
+        self.start_raylet_monitor()
+        if PY3 and self._ray_params.include_webui:
```

Comment:
Add a note here to tell reads the reason for starting dashboard in PYTHON3 only.